Remotion/Validation/Core/Implementation/DefaultValidatorFormatter.cs

```diff
@@ -31,8 +31,9 @@ public string Format (IPropertyValidator validator, Func<Type, string> typeNameF
     {
       ArgumentUtility.CheckNotNull ("validator", validator);
 
-      // TODO RM-7854: Assert that the returned value is not null.
-      return validator.ToString()!;
+      string? formattedValidator = validator.ToString();
+      Assertion.DebugIsNotNull (formattedValidator, "formattedValidator must not be null");
```

Comment:
```suggestion
      Assertion.IsNotNull (formattedValidator, "ToString() of validator type '{0}' returned null." validator.GetType());
```

Reasoning: validator is user-extensible, so you can't guarantee against somebody shooting at feet.